src/test/java/uk/gov/hmcts/reform/sscs/service/FurtherEvidenceServiceTest.java

```diff
@@ -407,6 +417,102 @@ public void givenDocList_shouldBeHandledUnderCertainConditions(List<SscsDocument
         assertEquals(expected, actual);
     }
 
+    @Test
+    public void updateCaseDocumentsWhichMatchCorrectCriteria() {
```

Comment:
would like a better description of this test method to describe what the test is checking.

Reply:
test name now more explicit